frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -17,21 +18,33 @@ const PVCStatus = ({pvc}) => {
 };
 
 const Header = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
-  <ColHead {...props} className="col-sm-4 hidden-xs" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-3 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-3 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="spec.resources.requests.storage">Requested</ColHead>
 </ListHeader>;
 
 const kind = 'PersistentVolumeClaim';
 const Row = ({obj}) => <div className="row co-resource-list__item">
-  <div className="col-sm-4 col-xs-6">
+  <div className="col-sm-3 col-xs-6">
     <ResourceLink kind={kind} name={obj.metadata.name} namespace={obj.metadata.namespace} title={obj.metadata.name} />
   </div>
-  <div className="col-sm-4 col-xs-4 co-break-word">
+  <div className="col-sm-3 col-xs-4 co-break-word">
     <ResourceLink kind="Namespace" name={obj.metadata.namespace} title={obj.metadata.namespace} />
   </div>
-  <div className="col-sm-4 hidden-xs">
-    <PVCStatus pvc={obj} />
+  <div className="col-sm-3 hidden-xs">
+    { obj.spec.volumeName ?
+      <React.Fragment>
+        <PVCStatus pvc={obj} /> to <Link to={`/k8s/cluster/persistentvolumes/${obj.spec.volumeName}`}>
```

Comment:
I know this is the design, but I personally feel like volume would be better as a separate column. If we use `ResourceLink`, it would have the `PV` badge, which is consistent with how we handle resource links in other tables. It also keeps the URL building logic in one place (`ResourceLink`).

Reply:
@julim@redhat.com

Reply:
From a UX perspective, our preference is to have the volume together with the status as per the design.

If we place the PVC in another column, it will take up more footprint, and also it may get confusing to the user as to the capacity being associated with the PV or PVC, which you could add a prefix to the Capacity label to make clearer, but that seems somewhat strange then.

+1 PVC badge as part of the PVC name (in the status column), to make it more consistent with elsewhere, so it would (and vice versa for the PVC listing):
[status_icon] Bound to [PVC_badge] my-claim

Thoughts @spadgett?

Reply:
I've just always found it much harder to understand tables that put multiple values in one column. It's harder scan down columns as effectively, values don't line up, and it add cognitive load. It also breaks things like column sorting, and I don't think we do it anywhere else in console currently.

I actually think layout would be simpler with separate columns. Sentences (even short ones) have a way of wrapping awkwardly in tables, and I'm not sure how we'd truncate long resource names. We'd need to be careful that the icon wraps with the resource name when things wrap.

I won't push too hard if you feel strongly on it, just my opinion. If we keep it as one column, we need to fix the CSS styles and make sure things wrap well and render well at mobile.

Reply:
I am actually hiding this information on XS screen

Reply:
@julim@redhat.com